src/app/file-manager/transferables-grid/transferables-grid.component.html

```diff
@@ -1,25 +1,46 @@
 <h1>Progress Status</h1>
 
-<p-toolbar styleClass="toolbar-accent">
-  <div class="ui-toolbar-group-left" style="width: 50%;">
-    <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12" style="padding: 0;">
-      <h4 class="subtitle" style="display: inline;">Downloading X of {{filesDatabase.data.length}} files</h4>
+<div class="flex-container space-between">
+  <div class="status-box-left">
+    <div class="flex-container space-between">
+      <h4 class="subtitle">Downloading X of {{filesDatabase.data.length}} files</h4>
       <h4 class="subtitle" style="float: right; text-align: right; display: inline;">X%</h4>
     </div>
+
     <p-progressBar mode="determinate" class="progress-bar" [value]="5" [showValue]="false"></p-progressBar>
+
     <p class="aclaration">
       <small>Estimated remaining time: </small>
     </p>
-  </div>
 
-  <div class="ui-toolbar-group-right" style="margin-top: 20px;">
-    <button mat-raised-button (click)="pauseSelected()" class="button-secondary">
-      <i class="material-icons">highlight_off</i>Cancel all</button>
-    <button mat-raised-button (click)="resumeSelected()" class="button-primary">
-      <i class="material-icons">pause_circle_outline</i>Pause all</button>
+    <div class="flex-container end actions">
+      <button mat-raised-button (click)="pauseSelected()" class="button-secondary">
+        <i class="material-icons">highlight_off</i>Cancel all</button>
+      <button mat-raised-button (click)="resumeSelected()" class="button-primary">
+        <i class="material-icons">pause_circle_outline</i>Pause all</button>
+    </div>
   </div>
-</p-toolbar>
 
+  <div class="status-box-right">
+    <div class="flex-container space-between">
+      <h4 class="subtitle">Uploading X of {{filesDatabase.data.length}} files</h4>
+      <h4 class="subtitle">X%</h4>
+    </div>
+
+    <p-progressBar mode="determinate" class="progress-bar" [value]="5" [showValue]="false"></p-progressBar>
+
+    <p class="aclaration">
+      <small>Estimated remaining time: </small>
+    </p>
+
+    <div class="flex-container end actions">
+      <button mat-raised-button (click)="pauseSelected()" class="button-secondary">
+        <i class="material-icons">highlight_off</i>Cancel all</button>
+      <button mat-raised-button (click)="resumeSelected()" class="button-primary">
+        <i class="material-icons">pause_circle_outline</i>Pause all</button>
+    </div>
+  </div>
+</div>
 
 <mat-expansion-panel #expa class="expand-panel">
 
```

Comment:
On line 48 below, need to update the text to read 'transfer list' instead of 'download list.